### PR TITLE
ZAP Chat subtitles for Video 16

### PR DESCRIPTION
**Pull request by 8linkz**
url: https://www.youtube.com/watch?v=_CKzFqDi33A

Please take a closer look at the following points:
-I don't understand line 77
-Line 78 I'm not sure about that
-09:40-09:45: Yiannis asks something -> I don't understand - subtitles missing

please check

### CODE REVIEW

**Comment by 8linkz** [comment]
The lifespan of videos decreases over time - so unfortunately this always has to be done at short notice :-(

**Comment by kingthorin** [comment]
Sorry for the delay. I'll try to review and listen tomorrow.

**Comment by kingthorin** [comment]
77/78 are actually correct.

**Comment by kingthorin** [comment]
The 9:40 part seems unimportant, I think he was just trying to agree/acknowledge.

**Review by kingthorin — APPROVED** [review]
I didn't have 30min to do the whole thing. But from what I read through and listened to it seems good.

**Comment by 8linkz** [comment]
> I didn't have 30min to do the whole thing. But from what I read through and listened to it seems good.

30 minutes is good - I usually sit there between 1-2 hours (just for proofreading and listening) :-)

**Comment by kingthorin** [comment]
I just said 30 because the vid is ~27 🤷‍♂️

Thanks for your efforts!!

File: subtitles/ZAP Chat/ZAP_Chat_16_Automation_Framework_Part_7_-_Authentication.srt [hunk 1]
```diff
@@ -0,0 +1,1330 @@
...
+00:16:16,760 --> 00:16:24,460
+I've done, so I've, using this authentication test context, I've added a request, which, so this
+
+227
+00:16:24,460 --> 00:16:31,200
+one was, we use it authenticated. So we've said the user, if you don't say which user, then it'll
+
+228
+00:16:31,200 --> 00:16:36,400
+be an authenticated request. We're very explicit about, you know, we do what we're told, or ZAP
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_16_Automation_Framework_Part_7_-_Authentication.srt]: unauthenticated

File: subtitles/ZAP Chat/ZAP_Chat_16_Automation_Framework_Part_7_-_Authentication.srt [hunk 2]
```diff
@@ -0,0 +1,1330 @@
...
+00:20:16,200 --> 00:20:22,300
+So if you change it back to auto-detect, then ZAP will auto-detect the right URL to verify every
+
+266
+00:20:22,300 --> 00:20:27,660
+time. And the same with session management. We're going to go back and change to auto-detect
+
+267
+00:20:27,660 --> 00:20:34,740
+session management. And we don't want it to be called authentication chest. We'll change it to
```
Comment (thc202) [inline subtitles/ZAP Chat/ZAP_Chat_16_Automation_Framework_Part_7_-_Authentication.srt]: chest → test

File: subtitles/ZAP Chat/ZAP_Chat_16_Automation_Framework_Part_7_-_Authentication.srt (hunk 1 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_16_Automation_Framework_Part_7_-_Authentication.srt]: done

File: subtitles/ZAP Chat/ZAP_Chat_16_Automation_Framework_Part_7_-_Authentication.srt (hunk 2 above)
Comment (8linkz) [reply subtitles/ZAP Chat/ZAP_Chat_16_Automation_Framework_Part_7_-_Authentication.srt]: done

**Review by thc202 — APPROVED** [review]

**Comment by thc202** [comment]
Thank you!

**Comment by psiinon** [comment]
Many thanks! Now applied and auto-translated into the usual set 😁

**Pull request merged**